app/models/event.rb

```diff
@@ -27,4 +37,15 @@ def set_times
       "All Day"
     end
   end
+
+  def index_image
+    variation =
+      ActiveStorage::Variation.new(Uploads.resize_to_fill(width: 220, height: 220, blob: image.blob, gravity: "Center"))
```

Comment:
@cdoyle-temple I need you to walk me through what is happening here. If I understand this correctly, you are generating the image to that specified size each time. Whch means if there are 10 images on a page, we are generating them 10 times.

Reply:
yes. is there a better way?